app/gateway/2.6.x/install-and-run/docker.md

```diff
@@ -0,0 +1,241 @@
+
+## Prerequisites
+
+For this installation, you'll need a Docker-enabled system with proper Docker access.
+
+## Install Gateway with a database
+
+1. **Pull the Docker image**
+
+   ```bash
+   docker pull kong/kong-gateway:{{include.kong_versions[13].version}}-alpine
+   ```
+
+   {:.important}
+   > Some [older {{site.base_gateway}} images](https://support.konghq.com/support/s/article/Downloading-older-Kong-versions)
+   are not publicly accessible. If you need a specific patch version and can't
+   find it on [Kong's public Docker Hub page](https://hub.docker.com/r/kong/kong-gateway), contact
+   [Kong Support](https://support.konghq.com/).
+
+   You should now have your {{site.base_gateway}} image locally.
+
+2. **Tag the image**
+
+   ```bash
+   docker tag kong/kong-gateway:{{include.kong_versions[13].version}}-alpine kong
+   ```
+
+3. **Create a Docker network**
+
+   Create a custom network to allow the containers to discover and communicate with each other.
+
+   ```bash
+   docker network create kong-net
+   ```
+
+4. **Start a database**
+
+   Start a PostgreSQL container:
+
+   ```p
+   docker run -d --name kong-database \
+     --network=kong-net \
+     -p 5432:5432 \
+     -e "POSTGRES_USER=kong" \
+     -e "POSTGRES_DB=kong" \
+     -e "POSTGRES_PASSWORD=kong" \
+     postgres:9.6
+   ```
+
+5. **Prepare the Kong database**
+
+   <pre><code>docker run --rm --network=kong-net \
+     -e "KONG_DATABASE=postgres" \
+     -e "KONG_PG_HOST=kong-ee-database" \
+     -e "KONG_PG_PASSWORD=kong" \
+     -e "KONG_PASSWORD=<div contenteditable="true">{PASSWORD}</div>" \
+     kong kong migrations bootstrap</code></pre>
+
+6. **Start Gateway with Kong Manager**
+
+   <pre><code>docker run -d --name kong --network=kong-net \
+     -e "KONG_DATABASE=postgres" \
+     -e "KONG_PG_HOST=kong-database" \
+     -e "KONG_PG_PASSWORD=kong" \
+     -e "KONG_PROXY_ACCESS_LOG=/dev/stdout" \
+     -e "KONG_ADMIN_ACCESS_LOG=/dev/stdout" \
+     -e "KONG_PROXY_ERROR_LOG=/dev/stderr" \
+     -e "KONG_ADMIN_ERROR_LOG=/dev/stderr" \
+     -e "KONG_ADMIN_LISTEN=0.0.0.0:8001" \
+     -e "KONG_ADMIN_GUI_URL=http://<div contenteditable="true">{HOSTNAME}</div>:8002" \
+       -p 8000:8000 \
+       -p 8443:8443 \
+       -p 8001:8001 \
+       -p 8444:8444 \
+       -p 8002:8002 \
+       -p 8445:8445 \
+       -p 8003:8003 \
+       -p 8004:8004 \
+       kong</code></pre>
+
+   {:.note}
+   > **Note**: The `HOSTNAME` for `KONG_PORTAL_GUI_HOST` should be preceded by a protocol, for example, `http://`.
+
+7. **Verify your installation**
+
+    Access the `/services` endpoint using the Admin API:
+
+    <pre><code>curl -i -X GET --url http://<div contenteditable="true">{HOSTNAME}</div>:8001/services</code></pre>
+
+    You should receive a `200` status code.
+
+    Verify that Kong Manager is running by accessing it using the URL specified in `KONG_ADMIN_GUI_URL`:
```

Comment:
```suggestion
  8. (Not available in OSS) Verify that Kong Manager is running by accessing it using the URL specified in `KONG_ADMIN_GUI_URL`:
```
I really wanted to put either "Enterprise only" or set the Enterprise badge, but ugh this is available in free mode, but only with the Enterprise image. Going with this phrasing as a stop-gap.